java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaSeekableByteChannel.java

```diff
@@ -135,7 +132,56 @@ protected MantaSeekableByteChannel(final AtomicReference<HttpUriRequest> request
         this.responseStream = responseStream;
         this.path = path;
         this.position = position;
-        this.connectionFactory = connectionFactory;
+        this.httpHelper = httpHelper;
```

Comment:
Can you checkout the Manta Hadoop Filesystem Driver and try to refactor it to the non-deprecated API as a sanity check?

Reply:
Checkout out hadoop-manta but it never touches the `MantaSeekableByteChannel` constructor. All access to this class is through `MantaClient#getSeekableByteChannel(String)` and `MantaSeekableByteChannel#position(long)`.